mindsdb/api/http/namespaces/databases.py

```diff
@@ -61,7 +61,14 @@ def post(self):
             )
 
         if check_connection:
-            handler = session.integration_controller.create_tmp_handler(name, database['engine'], parameters)
+            try:
+                handler = session.integration_controller.create_tmp_handler(name, database['engine'], parameters)
+            except ImportError as imort_error:
+                return http_error(
```

Comment:
Let do here the same as here: https://github.com/mindsdb/mindsdb/pull/10233/files#diff-5383daaa24b7faf0baa480508fd471d5c21b96d8b3e53fa1274930e7c64ce601R156
It won't be correct to return to the user 400 code, because it is not user's error. The best will be return 200 with relevant error message.